theano/tests/test_gradient.py

```diff
@@ -603,7 +603,6 @@ def test_subgraph_grad():
     for true_grad, pgrad in zip(true_grads, pgrads):
         assert(np.sum(np.abs(true_grad - pgrad)) < 0.00001)
 
-
```

Comment:
The line removed here should be added back; convention in python is two empty lines before a class declaration.